tests/end2end/features/hints.feature

```diff
@@ -174,6 +174,11 @@ Feature: Using hints
         # The actual check is already done above
         Then no crash should happen
 
+    Scenario: Error with invalid hint group
+        When I open data/hints/buttons.html
+        And I hint with args "INVALID_GROUP"
```

Comment:
Using "And I hint with args ..." automatically waits for the hints to appear before continuing. You want `And I run :hint INVALID_GROUP` here instead.